pryzm/assetlist.json

```diff
@@ -697,6 +697,53 @@
       ],
       "coingecko_id": "ondo-us-dollar-yield"
     },
+    {
+      "description": "Sky Protocol's Savings USDS",
+      "denom_units": [
+        {
+          "denom": "ibc/C9D6B804F68F218448D7D2AF5D26C0E6E93CD16F99D1695E77B5BE5CBD7D5ADB",
+          "exponent": 0
+        },
+        {
+          "denom": "susds",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/C9D6B804F68F218448D7D2AF5D26C0E6E93CD16F99D1695E77B5BE5CBD7D5ADB",
+      "name": "sUSDS",
+      "display": "susds",
+      "symbol": "sUSDS",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "cosmoshub",
+            "base_denom": "ibc/B9B561EB378C9EB8C13CAA11FCBC78E6B865A3C65707972F17B1052CFC39F473",
+            "channel_id": "channel-859"
+          },
+          "chain": {
+            "channel_id": "channel-0",
+            "path": "transfer/channel-0/transfer/08-wasm-1369/0xa3931d71877c0e7a3148cb7eb4463524fec27fbd"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/susds.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/susds.svg"
+      },
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "cosmoshub",
+            "base_denom": "ibc/B9B561EB378C9EB8C13CAA11FCBC78E6B865A3C65707972F17B1052CFC39F473"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/susds.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/susds.svg"
+        }
+      ],
+      "coingecko_id": "susds"
```

Comment:
Since this asset is bridged from Ethereum, it cannot share the same CoinGecko ID as the Ethereum asset. Please remove the coingecko ID here (unless there exists one specifically for the Eureka-bridged version)